stores/yaml/store_test.go

```diff
@@ -198,6 +198,15 @@ func TestEmpty2(t *testing.T) {
 }
 */
 
+func TestEmpty3(t *testing.T) {
+	branches, err := (&Store{}).LoadPlainFile([]byte("{}\n"))
+	assert.Nil(t, err)
+	assert.Equal(t, len(branches), 1)
+	bytes, err := (&Store{}).EmitPlainFile(branches)
+	assert.Nil(t, err)
+	assert.Equal(t, "{}\n", string(bytes))
+}
+
```

Comment:
This fails like this:
```
--- FAIL: TestEmpty3 (0.00s)
    store_test.go:206: 
                Error Trace:    store_test.go:206
                Error:          Expected nil, but got: &errors.errorString{s:"Error marshaling to yaml: yaml: expected SCALAR, SEQUENCE-START, MAPPING-START, or ALIAS, but got document end"}
                Test:           TestEmpty3
    store_test.go:207: 
                Error Trace:    store_test.go:207
                Error:          Not equal: 
                                expected: "{}\n"
                                actual  : ""
                            
                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -1,2 +1 @@
                                -{}
                                 
                Test:           TestEmpty3
FAIL
coverage: 69.5% of statements
FAIL    go.mozilla.org/sops/v3/stores/yaml      0.066s
FAIL
```

I wanted to demonstrate that in CI, but the test failed fast for #882.